### PR TITLE
Update item Settings in User Guide

### PR DESCRIPTION
**Pull request by TOsmanov**
### Motivation and context

This PR contains an update of the text and images of the Settings item.
### How has this been tested?

N/A
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/documentation/user_guide.md [hunk 1]
```diff
@@ -645,63 +645,57 @@ Applying ``split`` into an interpolated track divides it into two separate track
...
+-   Show ``Grid``, change grid size, choose color and transparency:
 
-    ![](static/documentation/images/image068.jpg)
+    ![](static/documentation/images/image068_mapillary_vistas.jpg)
 
--   Adjust ``Brightness``/``Contrast``/``Saturation`` of too exposing or too
-    dark images using ``F2`` — color settings (changes displaying and not the
+-   Show every image in full size or zoomed out like previous
+    (it is enabled by default for interpolation mode and disabled for annotation mode).
+- ``Rotate all images``  checkbox — switch the rotation of all frames /an individual frame.
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:663]:
```suggestion
- ``Rotate all images``  checkbox — switch the rotation of all frames or an individual frame.
```

File: cvat/apps/documentation/user_guide.md [hunk 2]
```diff
@@ -645,63 +645,57 @@ Applying ``split`` into an interpolated track divides it into two separate track
...
 
-![](static/documentation/images/image035.jpg)
+In tab ``Workspace`` you can:
 
----
-Go to the first and latest frames.
+![](static/documentation/images/image155.jpg)
 
-![](static/documentation/images/image036.jpg)
+- ``Enable Autosaving`` checkbox — turned off by default.
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:683]:
```suggestion
- ``Enable auto save`` checkbox — turned off by default.
```

File: cvat/apps/documentation/user_guide.md [hunk 3]
```diff
@@ -645,63 +645,57 @@ Applying ``split`` into an interpolated track divides it into two separate track
...
+- ``Autosaving Interval (min)`` input box — 15 minutes by default.
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:684]:
```suggestion
- ``Auto save interval (min)`` input box — 15 minutes by default.
```

File: cvat/apps/documentation/user_guide.md [hunk 4]
```diff
@@ -645,63 +645,57 @@ Applying ``split`` into an interpolated track divides it into two separate track
...
+- ``Show All Interpolation Tracks`` checkbox — shows hidden objects on the
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:685]:
```suggestion
- ``Show all interpolation tracks`` checkbox — shows hidden objects on the
```

File: cvat/apps/documentation/user_guide.md [hunk 5]
```diff
@@ -645,63 +645,57 @@ Applying ``split`` into an interpolated track divides it into two separate track
...
+- ``Attribute annotation mode (AAM) Zoom Margin`` input box — defines margins (in px) 
+  for shape in the attribute annotation mode.
+- Press `` Go back`` or ``F3`` to return to the annotation.
 
 ---
-Go to the next/previous frame with step equals to 1. Shortcuts: ``D`` — previous, ``F`` — next.
 
-![](static/documentation/images/image040.jpg)
 
 ---
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:700]: Should it be removed (we already have one above)?

File: cvat/apps/documentation/user_guide.md [hunk 6]
```diff
@@ -645,63 +645,57 @@ Applying ``split`` into an interpolated track divides it into two separate track
...
----
-Go to the next/previous frame with a predefined step. Shortcuts:
-``V`` — step backward, ``C`` — step forward. By default the step is ``10``
-(change at ``Open Menu`` —> ``Settings`` —> ``Player Step``).
+   ![](static/documentation/images/image152_DETRAC.jpg)
 
-![](static/documentation/images/image037.jpg)
+- ``Automatic bordering`` - enable automatic bordering for polygons and polylines during drawing/editing. 
+  For more information To find out more, go to the section [annotation with polygons](#Annotation-with-polygons).
+- ``Attribute annotation mode (AAM) Zoom Margin`` input box — defines margins (in px)
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:693]:
```suggestion
- ``Attribute annotation mode (AAM) zoom margin`` input box — defines margins (in px) 
```

File: cvat/apps/documentation/user_guide.md (hunk 5 above)
Comment (TOsmanov) [reply cvat/apps/documentation/user_guide.md:700]: @nmanovic these lines should be deleted at #1509. I will fix this by adding them back to this PR and deleting in #1509

**Pull request merged**